fashify/src/pages/ProductList/components/ProductListMain/ProductListMain.jsx

```diff
@@ -1,17 +1,20 @@
+import { useState } from "react";
 import { useData } from "../../../../contexts/data-context";
+import { useFilterHook } from "../../../../Hooks/FilterHook";
```

Comment:
Suggestion: use Barrel import if possibel

Reply:
Thanks for your suggestion